### DISCUSSION TITLE
Warnings

### DISCUSSION THREAD

**Discussion by roly151** [trickypr/bom-weather-docs#5] [Q&A] [answered]
Have you found an end point for Warning detail? I've tried a few with the warnings ID, but no luck yet

**Comment by trickypr** [comment]
No, but I will look into it and write up anything I find.

**Comment by trickypr** [comment, marked as answer]
You can access warning details with the `/warnings/<id>` endpoint where `id` is found through the [warning list api](https://trickypr.github.io/bom-weather-docs/docs/warnings#input). For example, https://api.weather.bom.gov.au/v1/warnings/NSW_MW001_IDN20400 is a marine weather warning that is currently active in Tathra.

**Reply by roly151** [reply]
Brilliant - thanks! I swear I tried that, but must have done something else.
